src/main/java/com/minecolonies/coremod/commands/colonycommands/RefreshColonyCommand.java

```diff
@@ -45,15 +53,61 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
         return super.getCommandUsage(sender) + "<ColonyId|OwnerName>";
     }
 
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final List<ActionArgument> actionArgumentList,
+            @NotNull final Map<String, Object> argumentValueByActionArgumentNameMap) throws CommandException
+    {
+        Colony colony = null;
+        final Object colonyObject = argumentValueByActionArgumentNameMap.get("colony");
+        if (null != colonyObject)
+        {
+            colony = (Colony) colonyObject;
+        }
+
+        if (null == colony)
+        {
+            EntityPlayer player = null;
+            final Object playerObject = argumentValueByActionArgumentNameMap.get("player");
+            if (null != playerObject)
+            {
+                player = (EntityPlayer) playerObject;
+                if (player != null)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2589)